flink-streaming-java/src/main/java/org/apache/flink/streaming/api/environment/StreamExecutionEnvironment.java

```diff
@@ -761,6 +768,20 @@ public void configure(ReadableConfig configuration, ClassLoader classLoader) {
 			});
 		config.configure(configuration, classLoader);
 		checkpointCfg.configure(configuration);
+		configuration.getOptional(ExecutionOptions.JOB_LISTENERS)
+			.ifPresent(l -> {
+				for (String listener : l) {
+					try {
```

Comment:
You can use  the method `InstantiationUtil#instantiate(final String className, final Class<T> targetType, final ClassLoader classLoader)`.

Reply:
I have used this tool